pallets/asset-handler/src/tests.rs

```diff
@@ -587,6 +587,28 @@ pub fn test_convert_amount_for_foreign_chain() {
 	});
 }
 
+use frame_support::traits::tokens::fungible::Mutate;
+
+#[test]
+pub fn test_withdrawal_fee() {
+	let (asset_address, _recipient, _sender, chain_id) = withdraw_data();
+	new_test_ext().execute_with(|| {
+		Balances::mint_into(&chainbridge::Pallet::<Test>::account_id(), 5000000000000);
+		assert_ok!(AssetHandler::withdraw_fee(Origin::signed(1), 3));
+		assert_eq!(
+			Balances::free_balance(&chainbridge::Pallet::<Test>::account_id()),
+			1000000000000
+		);
+		assert_eq!(Balances::free_balance(3), 4000000000000);
+		assert_ok!(AssetHandler::withdraw_fee(Origin::signed(1), 3));
+		assert_eq!(
+			Balances::free_balance(&chainbridge::Pallet::<Test>::account_id()),
+			1000000000000
+		);
+		assert_eq!(Balances::free_balance(3), 4000000000000);
+	});
+}
+
```

Comment:
It would be nice to have test which tests [this](https://github.com/Polkadex-Substrate/Polkadex/blob/07ea20d4247f0789211fb324dcac33c2cc3c5289/pallets/asset-handler/src/lib.rs#L439) assertion as well.